### ISSUE TITLE
Migrate to CSI

### ISSUE THREAD

**Issue by timoschwarzer** [timoschwarzer/btrfs-provisioner#13]

**Comment by tbrasser** [comment]
Found another dev that recently started a btrfs csi repo and I thought maybe connecting you can be fruitfull?

https://github.com/seungwoo-lee-518

**Comment by timoschwarzer** [comment]
@tbrasser thanks for letting me know! Unfortunately I was unable to find the btrfs CSI there, just this repository that doesn't seem to contain any code: https://github.com/seungwoo-lee-518/kubernetes-csi-btrfs/tree/feat/add-btrfs-functionality

Maybe I didn't search far enough.

This repository is currently more in a works-for-me state and I currently don't really have time to contribute to it. So should there be a working btrfs CSI somewhere I'd recommend rather use that. Otherwise I might still add CSI conformity to this project, but I cannot give an ETA.

**Comment by tbrasser** [comment]
Yeah sorry for not being clear, it's just the only other dev I found with seemingly similar intentions, not ready-to-use code. Probably notifying him about this repo would have made more sense.

And well, it's better than nothing, apparently tupperware (facebook) heavily uses btrfs for storage and it seems a shame nothing for k8s exists.

I'd gladly try your provisioner out at home and be active here, I don't expect any service/support and definately not on any strict timescale. ;)
